packages/teams-js/src/internal/utils.ts

```diff
@@ -342,12 +341,14 @@ export function base64ToBlob(mimeType: string, base64String: string): Promise<Bl
      *      constructor expects binary data.
      */
     if (mimeType.startsWith('image/')) {
+      const byteCharacters = atob(base64String);
       const byteArray = new Uint8Array(byteCharacters.length);
       for (let i = 0; i < byteCharacters.length; i++) {
         byteArray[i] = byteCharacters.charCodeAt(i);
       }
       resolve(new Blob([byteArray], { type: mimeType }));
     }
+    const byteCharacters = Buffer.from(base64String, 'base64').toString();
```

Comment:
Is there any validation other than a null check to ensure it is a valid base64 string? If not, it would be better to enclose the entire transformation within a try-catch block and reject it with an error in the catch block.

Reply:
We already have a unit test case for that in place.